TCMSApp/gulp.config.js

```diff
@@ -14,6 +14,7 @@ module.exports = function() {
         ignorePath: '../..'
     };
     var nodeModules = 'node_modules';
+    //var shell = require('gulp-shell');
```

Comment:
adding the commented line of code without any reason is a really bad practice :)